src/main/java/domain/Rank.java

```diff
@@ -0,0 +1,58 @@
+package domain;
+
+import java.util.Arrays;
+
+public enum Rank {
+	NOTHING(0, 0, false),
+	FIFTH(5000, 3, false),
+	FOURTH(50000, 4, false),
+	THIRD(150000, 5, false),
+	SECOND(30000000, 5, true),
+	FIRST(2000000000, 6, false);
```

Comment:
`_`를 추가하면 가독성에 더 좋아요!
```suggestion
	FIFTH(5_000, 3, false),
	FOURTH(50_000, 4, false),
	THIRD(150_000, 5, false),
	SECOND(30_000_000, 5, true),
	FIRST(2_000_000_000, 6, false);
```